english_style_guide.md

```diff
@@ -0,0 +1,113 @@
+_This document has been adapted from
+https://github.com/raywenderlich/english-style-guide_
+
+-------------------------------------------------------------------------------
+
+# English Style Guidelines
+
+This is an English style guide that Hack Club workshops should follow to stay
+consistent in how we refer to common things.
+
+**Table of Contents**
+
+- [English Style Guidelines](#english-style-guidelines)
+  - [Terminology and Capitalization](#terminology-and-capitalization)
+    - [General Guides](#general-guides)
+    - [List of Terminology](#list-of-terminology)
+  - [Punctuation](#punctuation)
+  - [Other Style Guidelines](#other-style-guidelines)
+
+## Terminology and Capitalization
+
+### General Guides
+
+**Company names** Company names are proper nouns and should be capitalized
+according to how the company capitalizes it's name.
+
+### List of Terminology
+
+**club leader** Not _Hack Club leader_ or _club director_.
+
+**console** Not _command line_, _command prompt_, or _terminal_.
+
+**double-click** Not _double click_.
+
+**filename**
+
+**Hack Club**
+
+**hacker**
+
+**OK** Don't use _Okay_ or _Ok_.
+
+**pop-up** Don't use _popup_ or _pop up_.
+
+**right-click**
+
+**URL**
+
+## Punctuation
+
+**Sentences End With Punctuation**
+
+All sentences should end with punctuation, even if they end with inline-code, a
+URL, or a filename.
+
+_Example:_
+
+_Correct: Go to https://google.com._
+
+_Correct: Make sure the css selector is set to `#username`._
+
+_Correct: Your website is now live at `example.com`!_
+
+_Incorrect: This will save to `index.html`_
+
+**Serial (Oxford) Commas**
+
+For simplicity the final comma in a list of elements (usually the one before the
+'and' or 'or') is left in the list.
+
+_Example:_
+
+_Correct: Make sure to include the HTML, CSS, and JS files._
+
+_Correct: This guide details standards for developing flexible, durable, and
+sustainable HTML and CSS._
+
+## Other Style Guidelines
+
+**bold**
+
+Titles that are to be omitted from a table of contents should be bolded instead
+of added to a header. Usually, repeated titles should be bolded.
+
+_Example:_
+
+```
+## Add an Image to The Website
+
+We can add an image to the site with an `<img>` tag. Unlike `<p>` tags, `<img>`
+tags don't have a closing tag.
+
+**Our code so far:**
+    <p>This is text</p>
+    <img href="example.com/icon.png"></img>
+```
+
+**italicize**
+
+Anything that needs emphasis should be italicized.
```

Comment:
Is this document in violation of this? It looks like we're using **bolding** for emphasis.